apps/bfd-server-ng/src/main/java/gov/cms/bfd/server/ng/claim/model/ClaimRelatedCondition.java

```diff
@@ -0,0 +1,27 @@
+package gov.cms.bfd.server.ng.claim.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.util.Optional;
+import org.hl7.fhir.r4.model.ExplanationOfBenefit;
+
+@Embeddable
+class ClaimRelatedCondition {
+
+  @Column(name = "clm_rlt_cond_sgntr_sqnc_num")
+  private long claimRelatedConditionSgnrNumber;
```

Comment:
We tend to prefer writing out the whole word here for those that may not know what `sgntr` means.

```suggestion
  private long claimRelatedConditionSignatureNumber;
```